app/models/manageiq/providers/openstack/cloud_manager.rb

```diff
@@ -30,7 +30,7 @@ class ManageIQ::Providers::Openstack::CloudManager < ManageIQ::Providers::CloudM
            :class_name  => "ManageIQ::Providers::StorageManager",
            :autosave    => true
 
-  include ManageIQ::Providers::Openstack::CinderManagerMixin
```

Comment:
@jerryk55 
Was this changed from `CinderManagerMixin` and you're changing it back?

Reply:
Yes but I'm having trouble finding where it was changed in the history.

Reply:
how does this help? The codes are identical..

Reply:
Oh good.  Then we can delete the identical code that was added under the Openstack Provider.  No need for duplicated code, right?